### DISCUSSION TITLE
Soc Fan?/?

### DISCUSSION THREAD

**Discussion by Broadmeadow-Tekton** [Guilouz/Creality-Helper-Script-Wiki#652] [Issues]
Just rooted my new K1 Max - as I did my first one about one month ago.  The only difference between the two is the last one now has a "Soc Fan" listed in the Temperature panel of Mainsail.  Don't know what that is for and can't find any references to it.  Can someone point me in the right direction?  I've included a screen grab within the included PDF for reference.
[soc fan.pdf](https://github.com/user-attachments/files/16638696/soc.fan.pdf)

Thank you.

**Comment by mtbivan** [comment]
inizialmente supponevo fosse la ventola dela scheda madre ma ora non ne sonno sicuro.
